includes/Widgets/Stories.php

```diff
@@ -545,7 +545,6 @@ private function input( array $args ) {
 			]
 		);
 
-		ob_start();
```

Comment:
I don't think you can remove this, as it breaks the `ob_get_clean()` and `label_before` / `echo $label;` logic below.

Looks like this was done here so that the label only needs to be written once, but can be echoed in two places, depending on `label_before`.

IMO we could also just do

```php
if ( $args['label_before'] ) {
?>
<label for="<?php echo $this->get_field_id( $args['id'] ); ?>">
	<?php echo esc_attr( (string) $args['label'] ); ?>
</label>
<?php
}

// ....

if ( ! $args['label_before'] ) {
?>
<label for="<?php echo $this->get_field_id( $args['id'] ); ?>">
	<?php echo esc_attr( (string) $args['label'] ); ?>
</label>
<?php
}
````

Reply:
My bad. Didn't see that. I have made this it's own method now.